docs/running-on-yarn.md

```diff
@@ -234,18 +234,11 @@ To use a custom metrics.properties for the application master and executors, upd
     The amount of off-heap memory (in megabytes) to be allocated per executor. This is memory that accounts for things like VM overheads, interned strings, other native overheads, etc. This tends to grow with the executor size (typically 6-10%).
   </td>
 </tr>
-<tr>
-  <td><code>spark.yarn.driver.memoryOverhead</code></td>
```

Comment:
I think we should make this configuration backward compatible, user should still be able to use `spark.yarn.driver.memoryOverhead` with warning log, like other deprecated configurations.

Reply:
Is there an example PR for deprecating a config property that I can follow?

Reply:
Look for `configsWithAlternatives` in `SparkConf.scala`.

Reply:
Ah, done.